APPX: Fix MS store certification error from appxmanifest.xml

starnight opened the pull request:
https://phabricator.endlessm.com/T34622

wjt commented on build_winapp/appxmanifest.xml:
```diff
@@ -1,14 +1,6 @@
-    Version="1.0.0.0"
-    Publisher="CN=4B001A25-B854-44D9-BAD3-8AD7FC7A8761"
-    ProcessorArchitecture="x64" />
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities" xmlns:uap3="http://schemas.microsoft.com/appx/manifest/uap/windows10/3" xmlns:desktop="http://schemas.microsoft.com/appx/manifest/desktop/windows10">
```
I find it quite hard to believe that Microsoft's validator is so broken that it can't handle newlines in an XML element. Did you test if this is actually necessary?

starnight replied:
Well, simply follow the format of appxmanifest.xml generated from Visual Studio.

starnight committed:
APPX: Bump TargetDeviceFamily's MinVersion to 10.0.17763.0

https://phabricator.endlessm.com/T34622

starnight committed:
APPX: Turn on Windows level Per monitor DPI awareness

The Windows App Certification Kit asks the DPI awareness feature. [1]
Endless Key app uses Electron as the frontend part. So, turn on Windows
level Per monitor DPI awareness feature in the appxmanifest.xml.

[1]: https://phabricator.endlessm.com/T34622#974326

https://phabricator.endlessm.com/T34622

wjt commented on build_winapp/appxmanifest.xml:
```diff
@@ -31,7 +31,8 @@
       Executable="kolibri-electron.exe"
-      EntryPoint="Windows.FullTrustApplication">
+      EntryPoint="Windows.FullTrustApplication"
+      xmlns="urn:schemas-microsoft-com:asm.v3">
```
```
MakeAppx : error: Error info: error C00CE014: App manifest validation error: The app manifest must be valid as per schema: Line 31, Column 6, Note: The schema for MaxVersionTested specified does not recognize XML fields with namespace "urn:schemas-microsoft-com:asm.v3". Please ensure that you have the correct MaxVersionTested specified. Reason: Element '{urn:schemas-microsoft-com:asm.v3}Application' is unexpected according to content model of parent element '{http://schemas.microsoft.com/appx/manifest/foundation/windows10}Applications'.
```

starnight replied:
Ya!  Trying to find out workaround to tell appcert that kolibri-electron.exe has DPI awareness feature.  But, seems this way is wrong.

starnight commented:
Looks like the DPI awareness issue is a big one.  I decided to close this PR.  Will have another PRs for each feature.
